test/e2e/policies.test-suite/balloons/n4c16/test10-health-checking/code.var.sh

```diff
@@ -0,0 +1,17 @@
+# Test that
+# Syntatically incorrect configuration file is resulting in the nri-resmgr
+# pod readiness probe failure.
+vm-put-file "${SRC_DIR}/test/e2e/policies.test-suite/balloons/n4c16/test10-health-checking/nri-resmgr-configmap.yaml" nri-resmgr-configmap.yaml
```

Comment:
We should avoid putting the full path here, it is just a pain later if changes are needed.
I think you can say here `vm-put-file $(instantiate nri-resmgr-configmap.yaml) nri-resmgr-configmap.yaml` the instantiate func should automatically figure out where the source file is.

Reply:
try by renaming the `nri-resmgr-configmap.yaml` to `nri-resmgr-configmap.yaml.in` in the file system, but do not add the .in in the script. The instantiate function will search the .in file and do any replacements (there are none in this case) and then sets the filename properly with path.

Reply:
that helped, thanks